packages/appstore/src/modules/traders-hub/account-with-wallets.tsx

```diff
@@ -1,29 +1,99 @@
 import React from 'react';
 import Wallet from 'Components/containers/wallet';
-import { observer } from '@deriv/stores';
-import { useWalletList } from '@deriv/hooks';
-import { Loading } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import WalletCardsCarousel from 'Components/wallet-cards-carousel';
+import { useContentFlag, useWalletList } from '@deriv/hooks';
+import { ButtonToggle } from '@deriv/components';
+import { convertWallets } from 'Constants/utils';
+import ButtonToggleLoader from 'Components/pre-loader/button-toggle-loader';
+import { routes } from '@deriv/shared';
+import WalletOptionsAndMultipliersListing from 'Components/wallet-content/wallet-option-multipliers-listing';
+import WalletCFDsListing from 'Components/wallet-content/wallet-cfds-listing';
 
 const AccountWithWallets = observer(() => {
-    const { data, isLoading } = useWalletList();
-    const [selected_wallet, setSelectedWallet] = React.useState<NonNullable<typeof data>[number]['loginid']>();
+    const {
+        ui: { is_mobile, is_dark_mode_on },
+        client: { is_landing_company_loaded, loginid, switchAccount },
+        traders_hub: { selected_platform_type, setTogglePlatformType, is_eu_user },
+    } = useStore();
 
-    if (isLoading) return <Loading is_fullscreen={false} />;
+    const { is_eu_demo, is_eu_real } = useContentFlag();
+    const eu_title = is_eu_demo || is_eu_real || is_eu_user;
+
+    const { data } = useWalletList();
+
+    const platform_toggle_options = [
+        { text: 'CFDs', value: 'cfd' },
+        { text: eu_title ? 'Multipliers' : 'Options & Multipliers', value: 'options' },
+    ];
+
+    const platformTypeChange = (event: {
+        target: {
+            value: string;
+            name: string;
+        };
+    }) => {
+        setTogglePlatformType(event.target.value);
+    };
+
+    // TODO: Will be deleted later because all needed information will be added to useWalletList hook
+    const wallet_accounts = React.useMemo(() => convertWallets(data, is_dark_mode_on), [data, is_dark_mode_on]);
+
+    const active_wallet_index =
```

Comment:
@farzin-deriv yes, we need this because carousel is using number of wallets array. I will try to move this code inside of `WalletCardsCarousel` but we definitely need this for pagination